lib/twilio-ruby/twiml/twiml.rb

```diff
@@ -26,13 +26,19 @@ class TwiML
         attr_accessor :name
 
         def initialize(**keyword_args)
+          @overrides = {
+              'aliasAttribute': 'alias',
```

Comment:
no need to wrap the keys in quotes if there are no special characters

(since the keys are symbols, not strings, the quotes are kind of confusing anyway)